config/test.json

```diff
@@ -1,5 +0,0 @@
-{
-  "security": {
-    "secret": "test"
-  }
-}
```

Comment:
Why has this file been removed? The integration tests now fail because there is no configured security secret in its environment.